airflow/jobs/scheduler_job.py

```diff
@@ -1741,6 +1730,16 @@ def _run_scheduler_loop(self) -> None:
             if not is_unit_test:
                 time.sleep(self._processor_poll_interval)
 
+            # Verify that all files were processed, and if so, deactivate DAGs that
+            # haven't been touched by the scheduler as they likely have been
+            # deleted.
+            if self.processor_agent.all_files_processed:
```

Comment:
This is fine on master, but will need updating under a world where multiple schedulers can coexist.

Would you mind leaving this change out for two reasons: 1. It's a separate issue to the pr subject 😀 2. To avoid conflicts on scheduler ha which I'm hopeful to merge today or tomorrow

Reply:
yeah sure @ashb.. will remove this code block from the PR for now, and will try to fix it next week.